desktop/ui/src/main/java/org/datacleaner/widgets/DCPopupBubble.java

```diff
@@ -34,8 +34,16 @@
 
 public class DCPopupBubble {
 
+    public interface PopupCallback {
```

Comment:
That is simply not true for interfaces. The "static" modifier makes no sense here.